dictionaries/hr_HR/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "@cspell/dict-hr-hr",
-  "version": "1.0.8",
-  "description": "Croatian dictionary for cspell.",
+  "version": "1.0.0",
```

Comment:
We need to keep the version as `1.0.8` to so it will get correctly updated to the next version when publishing.